lib/statsd/class-statsd.php

```diff
@@ -15,8 +15,25 @@ class StatsD {
 	 * @param string|array $stats The metric(s) to set.
 	 * @param float $time The elapsed time (ms) to log
 	 **/
-	public static function timing( $stats, $time ) {
-		self::update_stats( $stats, $time, 1, 'ms' );
+	public function timing( $stats, $time ) {
+		$this->update_stats( $stats, $time, 1, 'ms' );
+	}
+
+	/**
+	 * Tracks time of callback and then sets one or more timing values
+	 *
+	 * @param string|array $stats The metric(s) to set.
+	 * @param callback $callback to be tracked
+	 **/
+	public function time( $stats, $callback ) {
```

Comment:
It doesn't look like this function is actually used anywhere, so probably better to leave it out of this PR (until it's actually needed).

Reply:
Good point.